framework/include/timesteppers/TimeStepper.h

```diff
@@ -133,7 +133,7 @@ class TimeStepper : public MooseObject, public Restartable, public ScalarCouplea
   Real & _timestep_tolerance;
 
   ///should detailed diagnostic output be printed
-  bool & _verbose;
+  const bool & _verbose;
```

Comment:
Minor: This doesn't need to be a reference anymore - the assumption is that it's const.